src/lib/flexbox/api/flex.ts

```diff
@@ -301,8 +311,21 @@ export class FlexDirective extends BaseFxDirective implements OnInit, OnChanges,
     let usingCalc = String(basis).indexOf('calc') > -1;
     let isPx = String(basis).indexOf('px') > -1 || usingCalc;
 
-    css[min] = (basis == '0%') ? 0 : isPx ? basis : null;
-    css[max] = (basis == '0%') ? 0 : usingCalc ? null : basis;
```

Comment:
Done.
Though it does look a bit awkward. 

No Problem. 
Thanks for looking over so quick.